test/test_cli_integration.py

```diff
@@ -28,10 +28,11 @@ class CliIntegrationTest(unittest.TestCase): # pragma: no cover
         'CONJUR_HTTPS_APPLIANCE_URL': 'url',
     }
 
-    HTTPS_CA_BUNDLE_ENV_VAR = { 'CONJUR_CA_BUNDLE': 'ca-bundle' }
+    HTTPS_CA_BUNDLE_ENV_VAR = {'CONJUR_CA_BUNDLE': 'ca-bundle'}
 
     DEFINED_VARIABLE_ID = 'one/password'
-
+    DEFINED_VARIABLE2 = 'two/username'
```

Comment:
i would give it more meaningful names

Reply:
I actually don't think it is so critical for tests...
What name would you use?

Reply:
I like uniformity so I would write `DEFINED_VARIABLE_USERNAME` or  `DEFINED_VARIABLE_TWO`. It is just cleaner for the 👁️